src/components/TodoElement.tsx

```diff
@@ -0,0 +1,125 @@
+import React, { useEffect, useRef, useState } from 'react';
+import cn from 'classnames';
+
+import { Todo } from '../types/Todo';
+
+interface Props {
+  todo: Todo;
+  removeTodo: (id: number) => void;
+  loadingTodo: number[];
+  updateStatus: (id: number) => void;
+  updateTitle: (id: number, title: string) => void;
+}
+
+export const TodoElement: React.FC<Props> = ({
+  todo,
+  removeTodo,
+  loadingTodo,
+  updateStatus,
+  updateTitle,
+}) => {
+  const { id, title, completed } = todo;
+  const [editValue, setEditValue] = useState(title);
+  const [editForm, setEditForm] = useState(false);
+
+  const ref = useRef<HTMLInputElement>(null);
+
+  useEffect(() => {
+    if (editForm && ref.current) {
+      ref.current.focus();
+    }
+  }, [editForm]);
+
+  const apply = () => {
+    updateTitle(id, editValue);
+    setEditForm(false);
+  };
+
+  const handleDoubleClick = () => {
+    setEditForm(true);
+  };
+
+  const handleBlur = () => {
+    apply();
+  };
```

Comment:
When I delete the title and click outside, title don't change and todo set to completed 

![image](https://github.com/mate-academy/react_todo-app-with-api/assets/84296041/5ad09b3f-7c2b-4e0e-8011-25af5a3db1f0)
![image](https://github.com/mate-academy/react_todo-app-with-api/assets/84296041/13c3b4a4-5dff-4cc1-83e4-8bd1d79fab81)


And after when I open it, the input field is empty

![image](https://github.com/mate-academy/react_todo-app-with-api/assets/84296041/d41c90a3-e8dd-4177-b279-63e9237366f1)